diff --git a/sound/squeezelite/Makefile b/sound/squeezelite/Makefile
index 9c8fac2..ce90c94 100644
--- a/sound/squeezelite/Makefile
+++ b/sound/squeezelite/Makefile
@@ -66,7 +66,7 @@ define Package/squeezelite-custom
 endef
 
 define Package/squeezelite-custom/config
-	if PACKAGE_squeezelite-custom
+	menu "Squeezelite custom configuration"
 		config SQUEEZELITE_AAC
 		    bool "AAC codec support"
 		    help
@@ -132,7 +132,7 @@ define Package/squeezelite-custom/config
 		    help
 			WMA and ALAC codec support
 		    default n
-	endif
+	endmenu
 endef
 
 define Package/squeezelite/description/default
